example/src/containers/Dashboard/Dashboard.tsx

```diff
@@ -22,6 +24,25 @@ export const Dashboard = () => {
     let [publicKey, setPublicKey] = useState("");
     let [latestBlock, setLatestBlock] = useState<BlockInfo>({hash: "", number: ""});
 
+    let [api, setApi] = useState<PolkadotApi>(null);
+
+    const balanceCallback = (...args: unknown[]) => {
```

Comment:
@mpetrunic I have a problem binding this function so it actually works when called to remove the listener. Could you give me some directions?

Reply:
@MakMuftic https://reactjs.org/docs/hooks-reference.html#usecallback